raiden/tests/utils/transfer.py

```diff
@@ -277,18 +276,13 @@ def assert_synced_channel_state(
     assert_mirror(channel1, channel0)
 
 
-def wait_assert(*args, func, timeout=5, **kwargs):
+def wait_assert(func, *args, **kwargs):
     """Like assert_*, but retry and raises the last AssertionError only after timeout"""
```

Comment:
You may want to take out the `timeout` part of the docstring then

Reply:
fixed